bin/riemann-dir-files

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+# Gets the number of files present on a directory and submits it to riemann
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+
+class Riemann::Tools::DirFiles
```

Comment:
Yeah, I could see "DirFiles" meaning any number of things, so I'd include the phrase "Count" somewhere in the name. FileCount, DirCount, DirFilesCount, something like that. :)